Makefile

```diff
@@ -78,3 +78,8 @@ submodules-init:
 .PHONY: submodules-deinit
 submodules-deinit:
 	git submodule deinit -f --all
+
+# Install extension
+.PHONY: install-extension
+install-extension:
+	cd editor/code && npm i && npm run compile
```

Comment:
This doesn't really "install" the extension: it simply builds it.

Reply:
`npm i` installs it no?

Reply:
No, it just downloads the `npm` modules specified in `package.json` to `node_modules`.

Reply:
OK renamed to "extension"